### PR TITLE
css repair 1

### PR DESCRIPTION
**Pull request by lawrence1357**

### CODE REVIEW

File: spec/support/test_helpers.rb [hunk 1]
```diff
@@ -6,10 +6,12 @@ def login_user
     Capybara.current_session.driver.browser.clear_cookies
     User.delete_all
     @user = User.create(email: 'email@berkeley.edu', password: 'password')
+    # @user.skip_confirmation!
```
Comment (alice-wang) [inline spec/support/test_helpers.rb]: delete the commented lines if they're unnecessary

File: app/assets/stylesheets/typeheadjs1.css [hunk 2]
```diff
@@ -103,6 +103,15 @@ html {
...
+}
+
+body {
+    padding: 0;
+    margin: 0;
+    overflow: hidden;
+    height: 100%;
+    width: 100%;
+}
 /*    background-color: rgba(192,192,192,0.8);
```
Comment (alice-wang) [inline app/assets/stylesheets/typeheadjs1.css]: Delete unnecessary lines

File: spec/requests/studygroup_new.rb [hunk 3]
```diff
@@ -11,7 +12,36 @@ def fill_in_form
...
+    visit ('/studygroups/new')
+    fill_in('groupname', :with => "anything")
+ 	
+ 	select "3", :from => "start_hours"   
+ 	select "30", :from => "start_minutes"
+ 	select "A.M.", :from => "start_time_tag"  
+ 	select "4", :from => "end_hours"
+ 	select "30", :from => "end_minutes"
+ 	select "P.M.", :from => "end_time_tag" 
+    #select '2011/01/01', :from => "date"
```
Comment (alice-wang) [inline spec/requests/studygroup_new.rb]: delete commented lines

File: spec/support/test_helpers.rb [hunk 4]
```diff
@@ -6,10 +6,12 @@ def login_user
...
+    # @user.save
```
Comment (alice-wang) [inline spec/support/test_helpers.rb]: same hizere

File: app/views/studygroups/new.html.erb [hunk 5]
```diff
@@ -134,7 +134,7 @@
 
     <div id="the-basics">
       <%= label_tag :course %><br>
-      <input class="typeahead" type="text" name="course" placeholder="What Course?">
+      <input class="typeahead" type="text" name="course" placeholder="What Course?" id="hahaha">
```
Comment (dkumar) [inline app/views/studygroups/new.html.erb]: change the id

File: app/views/studygroups/new.html.erb [hunk 6]
```diff
@@ -187,6 +187,12 @@
     document.getElementById("minsize").defaultValue = 1;
     document.getElementById("maxsize").defaultValue = 6;
 
+    function isValidDate(s) {
```
Comment (dkumar) [inline app/views/studygroups/new.html.erb]: delete this if not needed

File: app/views/studygroups/new.html.erb [hunk 7]
```diff
@@ -196,8 +202,8 @@
         }
 
         var date = document.forms["add"]["date"].value;
-        if (date == null || date == "") {
-            errors.push("Please Enter a Date");
+        if (date == null || date == "" || isValidDate(date)) {
```
Comment (dkumar) [inline app/views/studygroups/new.html.erb]: remove call to `isValidDate`

File: app/views/studygroups/new.html.erb [hunk 8]
```diff
@@ -242,10 +248,13 @@
 
         if (errors.length > 0){
             alert(errors[0]);
+            console.log("error json: ", errors[0])
```
Comment (dkumar) [inline app/views/studygroups/new.html.erb]: Remove this

File: app/views/studygroups/new.html.erb [hunk 9]
```diff
@@ -242,10 +248,13 @@
...
             return false;
         }
         else{
             return true;
         }
+
```
Comment (dkumar) [inline app/views/studygroups/new.html.erb]: Unnecessary lines

File: spec/requests/studygroup_new.rb [hunk 10]
```diff
@@ -1,18 +1,95 @@
 require 'rails_helper'
 require_relative '../support/test_helpers'
-require 'simplecov'
-SimpleCov.start 'rails'
+# require 'simplecov'
+require 'spec_helper'
+Capybara.javascript_driver = :poltergeist
+# SimpleCov.start 'rails'
```
Comment (dkumar) [inline spec/requests/studygroup_new.rb]: Remove commented lines if unnecessary

File: test/test_helper.rb [hunk 11]
```diff
@@ -2,7 +2,7 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require 'simplecov'
-SimpleCov.start
+# SimpleCov.start
```
Comment (dkumar) [inline test/test_helper.rb]: Remove commented lines if unnecessary

**Pull request merged**